android/app/src/main/java/org/uwaterloo/subletr/theme/SubletrTypography.kt

```diff
@@ -128,4 +128,12 @@ val SubletrTypography = Typography(
 		letterSpacing = 0.sp,
 		textAlign = TextAlign.Center
 	),
+	displaySmall = TextStyle(
+		color = Color.Gray,
```

Comment:
I think we do need to change all of these to `secondaryTextColor` at some point but we can do that later ^-^